manifests/base/manager/controller_manager_config.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: config.trainer.kubeflow.org/v1alpha1
+kind: Configuration
+# Health configuration
+health:
+  healthProbeBindAddress: :8081
+  readinessEndpointName: readyz
+  livenessEndpointName: healthz
+
+# Metrics configuration
+metrics:
+  bindAddress: :8443
+  secureServing: true
+
+# Webhook configuration
+webhook:
+  port: 9443
+  host: ""
+
+# Leader election configuration
+leaderElection:
+  leaderElect: true
```

Shall we use enable: true, similar to cert manager ?
```suggestion
  enable: true
```